spread.yaml

```diff
@@ -8,17 +8,30 @@
 project: spread
 
 environment:
-    GOVERSION: 1.7.3
-    GOPATH: /home/test
-    PATH: $GOPATH/bin:$PATH
+    GOVERSION: 1.10.4
+    GOHOME: /home/test
+    GOPATH: $GOHOME
+    PATH: $GOHOME/bin:$PATH
     CACHE_DISABLED: $(HOST:[ "$SEND_CACHE" = 1 ] && echo false || echo tests/cache)
 
 backends:
+    google:
+        key: "$(HOST: echo $SPREAD_GOOGLE_KEY)"
+        location: computeengine/us-east1-b
+        halt-timeout: 2h
+        systems:
+            - ubuntu-18.04-64:
+                image: ubuntu-1804-64-virt-enabled
+                workers: 1
+
     qemu:
         systems:
             - ubuntu-16.04-64:
                 username: ubuntu
                 password: ubuntu
+            - ubuntu-18.04-64:
```

Comment:
Are both systems still necessary? If you tested with the latter, we can probably drop the former.

Reply:
No, I'll leave 18.04, thanks